backend/src/routes/students.js

```diff
@@ -61,6 +61,10 @@ router.get('/:id', async (req, res) => {
 
 /* POST students listing */
 router.post('/', async (req, res) => {
+  // If EnglishProficiency is empty
+  if (req.body.EnglishProficiency === "" || null) {
```

Comment:
Due to [logical operator precedence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence), `if (string === "something" || null)` is parsed as `(if string === "something") OR (if null)`, whereas we want `if string === ("something" OR null)`

```suggestion
  if (!req.body.EnglishProficiency || req.body.EnglishProficiency === "") {
```